File: programacao/c/respostas/operacoes/23.c
#include <stdio.h>

int main() {
    float preco_cobre, preco_zinco, massa_latao;
    float massa_cobre, massa_zinco, custo_total;

    printf("Digite o preco do kg de cobre (R$): ");
    scanf("%f", &preco_cobre);
    
    printf("Digite o preco do kg de zinco (R$): ");
    scanf("%f", &preco_zinco);
    
    printf("Digite a massa total de latao desejada (kg): ");
    scanf("%f", &massa_latao);

    massa_cobre = massa_latao * 0.70; // 70% da massa total
    massa_zinco = massa_latao * 0.30; // 30% da massa total

    custo_total = (massa_cobre * preco_cobre) + (massa_zinco * preco_zinco);

    printf("Para produzir %.2f kg de latao, serao necessarios:\n", massa_latao);
    printf(" - %.2f kg de cobre\n", massa_cobre);
    printf(" - %.2f kg de zinco\n", massa_zinco);
    printf("Custo total da producao: R$ %.2f\n", custo_total);

    return 0;
}

File: programacao/python/respostas/operacoes/23.py
preco_cobre = float(input("Preco do kg de cobre (R$): "))
preco_zinco = float(input("Preco do kg de zinco (R$): "))
massa_latao = float(input("Massa total de latao desejada (kg): "))

massa_cobre = massa_latao * 0.70
massa_zinco = massa_latao * 0.30
custo_total = massa_cobre * preco_cobre + massa_zinco * preco_zinco

print(f"Para produzir {massa_latao:.2f} kg de latao, serao necessarios:")
print(f" - {massa_cobre:.2f} kg de cobre")
print(f" - {massa_zinco:.2f} kg de zinco")
print(f"Custo total da producao: R$ {custo_total:.2f}")
